frontend/src/pages/Settings/Settings.tsx

```diff
@@ -83,6 +83,7 @@ const Settings: React.SFC<RouteComponentProps> = (props) => {
     axios
       .get(`${url}/users`, headers)
       .then(({ data }) => {
+        console.log(data);

```

Comment:
Can you remove this please?

Reply:
I rebased the whole thing so all these things should be fixed.